File: Crazy8s/Crazy8s/Crazy8ProvidedCode/Card (2016_10_11 04_07_57 UTC).js
"use strict";

/**
 * A single playing card.
 */
function Card(aSuit, aValue) {
  this.suit = aSuit;
  this.value = aValue;
}
Card.prototype = {
  getSuit: function () { return this.suit; },
  getValue: function () { return this.value; },
  /**
   * Return a string representation of this card.
   */
  toString: function () {
    return this.value + this.suit ;
  }
};

File: Crazy8s/Crazy8s2Player/ROOT/WEB-INF/classes/Card (2016_10_11 04_07_57 UTC).java
package Model;
import Model.*;
/**
 * A single playing card.
 */
public class Card {
  private String suit;
  private String value;
  public Card(String aSuit, String aValue)
  { suit = aSuit; value = aValue; }
  public String getSuit() { return suit; }
  public String getValue() { return value; }
  
  /**
   * Return a string representation of this card.
   * This is what will be printed if a Card is passed
   * as an argument to System.out.println().
   */
  public String toString() {
    return value + suit;
  }
}
